/**************************************************************************
 * basf2 (Belle II Analysis Software Framework)                           *
 * Author: The Belle II Collaboration                                     *
 *                                                                        *
 * See git log for contributors and copyright holders.                    *
 * This file is licensed under LGPL-3.0, see LICENSE.md.                  *
 **************************************************************************/
--- ../../cdc/simulation/src/SensitiveDetector.cc	2012-02-10 11:46:08.000000000 +0900
+++ ../../cdc/simulation/src/SensitiveDetector.cc.new	2012-02-10 11:46:03.000000000 +0900
@@ -104,7 +104,9 @@
     return true;
   }
 
+#ifdef CDC_ELOSS
   if (edep == 0.) return false;
+#endif
   if (charge == 0.) return false;
 
   // Calculate cell ID
